bin/forklift

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
+
+# Needs at least python 3.4 or the pathlib library
```

Comment:
I just want t make sure this is usable on most developers systems and if not out of the box, document what a user would need to do.